### ISSUE TITLE
Sync fail and unable to select .gpx file for import on one device but works fine on another device

### ISSUE THREAD

**Issue by dorjanbenic** [osmandapp/OsmAnd-iOS#3190]
I run iOS OsmAnd app on both my Mac and my iPhone, I use many apps like that and it works great usually. I had quite few crashes that day even on OsmAnd app on iPhone when I was managing Favourites. Since I got one update (4.5.2) and I haven’t have any issues since and everything else works except issues mentioned below.

OsmAnd for iOS app on iPhone
1. Syncs successfully (checked on web)
2. Unable to import .gpx file (files are greyed out, can’t select the file)

OsmAnd for iOS app on Mac:
1. Fails to sync (app crashes)
2. Able to imports .gpx file (file is selectable in file picker)

I have reinstalled OsmAnd on Mac but issue was not resolved. All of that happened on iOS 16.7.1 and MacOS 13.6. I have since update to iOS 17.1 and MacOS 14.1 but issue still persists.

<img width="1006" alt="Screenshot 2023-10-19 at 07 38 33" src="https://github.com/osmandapp/OsmAnd-iOS/assets/39588255/b21429a8-909f-413c-81b6-c057c310d35e">

**Comment by dmpr0** [comment]
We have issues for issues with Import https://github.com/osmandapp/OsmAnd-iOS/issues/2507

**Comment by vshcherb** [comment]
To do: We need to double check with OsmAnd Cloud sync and reproduce it.

**Comment by dorjanbenic** [comment]
This issue can be closed now. I have solved both issue, explanation below.

After upgrading to MacOS 14.1 I have deleted the OsmAnd and navigated to every folder in /Library and  /Username/Library that has OsmAnd it its name and deleted all of those files. Then I have restarted my Mac and installed OsmAnd app again. During app setup app have crashed once but after that everything started to work.

My response on issue Import [#2507](https://github.com/osmandapp/OsmAnd-iOS/issues/2507).

After update of iOS (17.1) on my iPhone 13 I have reinstalled latest version of OsmAnd, 4.5.5 i think. Then I have tried to import GPX file again (one I exported from OsmAnd) and it worked. Now every GPX tile I browse to in iOS file picker is not greyed out any more.

Not sure if the issue was resolved by updating iOS or by reinstalling OsmAnd app.

I guess some files remain on iOS (cache/configuration file) that is not deleted by deleting the app. Probably iOS upgrade process deletes some temporary files so new installation of OsmAnd is clean. On MacOS I can manually delete cache/configuration files but not on iOS.
